### PR TITLE
GUACAMOLE-269: Add fields for controlling Backspace behavior.

### PR DESCRIPTION
**Pull request by necouchman**
Client-side changes to allow the backspace key to be configured for SSH and Telnet sessions.

### CODE REVIEW

**Comment by necouchman** [comment]
@mike-jumper Not trying to bug, but just a reminder that this PR goes with the recently-merged changes on the server side for backspace key config.

File: guacamole/src/main/webapp/translations/en.json [hunk 1]
```diff
@@ -447,6 +448,9 @@
         "FIELD_HEADER_TYPESCRIPT_NAME" : "Typescript name:",
         "FIELD_HEADER_TYPESCRIPT_PATH" : "Typescript path:",
 
+        "FIELD_OPTION_BACKSPACE_8"    : "Backspace (Ctrl-H)",
+        "FIELD_OPTION_BACKSPACE_127"  : "Delete (Ctrl-?)",
```
Comment (mike-jumper) [inline guacamole/src/main/webapp/translations/en.json]: These options are swapped compared to the telnet options. Which is correct?

Comment (necouchman) [reply guacamole/src/main/webapp/translations/en.json]: Fixed.  SSH is correct - ASCII 8 is "Backspace", ASCII 127 is Del.

http://www.physics.udel.edu/~watson/scen103/ascii.html

Comment (mike-jumper) [reply guacamole/src/main/webapp/translations/en.json]: You'll also need an entry for the `""` value (`FIELD_OPTION_BACKSPACE_EMPTY`).

Comment (necouchman) [reply guacamole/src/main/webapp/translations/en.json]: Added.

**Pull request merged**